tests/test_s3/test_s3_logging.py

```diff
@@ -261,3 +268,301 @@ def test_log_file_is_created():
     assert any(c for c in contents if bucket_name in c)
     assert any(c for c in contents if "REST.GET.BUCKET" in c)
     assert any(c for c in contents if "REST.PUT.BUCKET" in c)
+
+
+@mock_s3
+def test_invalid_bucket_logging_when_permissions_are_false():
+    s3_client = boto3.client("s3", region_name=DEFAULT_REGION_NAME)
+    bucket_name = "mybucket"
+    log_bucket = "logbucket"
+    s3_client.create_bucket(Bucket=bucket_name)
+    s3_client.create_bucket(Bucket=log_bucket)
+    with patch(
```

Comment:
Patching like this will not work in ServerMode. You can skip these tests if you like:
```
from moto import settings
from  unittest import SkipTest
if settings.TEST_SERVER_MODE:
    raise SkipTest("Can't patch permission logic in ServerMode")
```